packages/shared-ux/chrome/navigation/src/ui/components/navigation_item.tsx

```diff
@@ -74,6 +75,10 @@ function NavigationItemComp<
 
   if (isRootLevel) {
     const href = getNavigationNodeHref(navNode);
+    const dataTestSubj = classNames(`nav-item`, {
```

Comment:
maybe we can unify this into `getNavItemTestSubj` ? But I didn't want to do this as part of this pr

Reply:
What difference would that bring?

Reply:
it is in 3 or 4 different places, but seems mostly the same.